bmi_lstm.py

```diff
@@ -300,148 +308,169 @@ def get_output_var_names(self):
-        """Set model values.
-
-        Parameters
-        ----------
-        var_name : str
-            Name of variable as CSDMS Standard Name.
-        src : array_like
-              Array of new values.
-        """ 
-        var_name = self.get_var_name( long_var_name )
-        setattr( self, var_name, value )
-
-        # jmframe: this next line is basically a duplicate. 
-        # I guess we should stick with the attribute names instead of a dictionary approach. 
-        self.all_lstm_input_values_dict[var_name] = value
-
-    #------------------------------------------------------------ 
-    def get_component_name(self):
-        """Name of the component."""
-        return self._name
-
-    #------------------------------------------------------------ 
-    def get_input_item_count(self):
-        """Get names of input variables."""
-        return len(self._input_var_names)
-
-    #------------------------------------------------------------ 
-    def get_output_item_count(self):
-        """Get names of output variables."""
-        return len(self._output_var_names)
-
-    #------------------------------------------------------------ 
-    def set_value_at_indices(self, name, inds, src):
-        """Set model values at particular indices.
+    #-------------------------------------------------------------------
+    # BMI: Variable Getter and Setter Functions
+    #-------------------------------------------------------------------
+    #-------------------------------------------------------------------
+    #def get_value(self, var_name, dest):
+    def get_value(self, var_name):
+        """Copy of values.
         Parameters
         ----------
         var_name : str
             Name of variable as CSDMS Standard Name.
-        src : array_like
-            Array of new values.
-        indices : array_like
-            Array of indices.
+        dest : ndarray
+            A numpy array into which to place the values.
+        Returns
+        -------
+        array_like
+            Copy of values.
         """
-        val = self.get_value_ptr(name)
-        val.flat[inds] = src
+        #dest[:] = self.get_value_ptr(var_name).flatten()
+        #return dest
+        return self.get_value_ptr(var_name) #JG Edit
 
-    #------------------------------------------------------------ 
-    def get_var_nbytes(self, var_name):
-        """Get units of variable.
+    #-------------------------------------------------------------------
+    def get_value_ptr(self, var_name):
+        """Reference to values.
         Parameters
         ----------
         var_name : str
             Name of variable as CSDMS Standard Name.
         Returns
         -------
-        int
-            Size of data array in bytes.
+        array_like
+            Value array.
         """
-        return self.get_value_ptr(var_name).nbytes
+
+        # JG Edit Note: This should be revisted, more of a band-aid rn
```

Comment:
I  came up with this:

        if getattr(self, var_name) != self._values[var_name]:
            print("WARNING: The variable ({}) stored in two locations is inconsistent".format(var_name))
        
        return getattr(self, var_name)   # We don't need to store the variable in a dict and as attributes
        return self._values[var_name]   # Pick a place to store them and stick with it.